modules/ROOT/pages/release_notes.adoc

```diff
@@ -0,0 +1,87 @@
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 0.10:
+
+
+* Triggers v 0.2.1:  Support added for multiple `triggerbindings`, user configurable `interceptors`, `jsonpath` in `triggerbindings`, labeling generated resources and dynamic client for custom resources in `eventlistener`.
+
+* Pipeline Templates: Adds pipeline templates which will be available to a user for reuse through the *+Add* flows in the Developer perspective of the OpenShift web console.
+
+* Script Enhancements: Multiple enhancements made to the script mode when defining task steps:
+** Task steps in script mode no longer need to specify a shebang line. Scripts that do not start with shebang, by default, will have a value of `#!/bin/sh`
+** Support for arguments in script mode
+
+** Support for script mode for Conditions
+
+** Script mode now inserts `set -xe` to scripts that don’t include a shebang `#!`.
+
+* Support for workspace in `Task`, `TaskRun`, `Pipeline` and `PipelineRun` to simplify sharing files between tasks in a pipeline
+
+* Enhancements to `PipelineResource` to support unused resources.
+
+* Support for `taskSpec` with `pipelineSpec` to support embedding tasks inside a pipeline
+
+* Task spec now supports results and handles all changes in the entrypoint code.
+* Support for `from` in Pipeline Conditions. Resources in Pipeline Conditions can now declare that they depend on the output of previous tasks using the `from` clause
+* Support for `sslverify` flag for Pipeline Resource of type `git`
+* A description field is now added to `ResourceDeclaration`.
+* The default `app.kubernetes.io/managed-by` label value is now configurable in a ConfigMap
+* Support for GitLab on the PR resource.
+
+
+== Getting support
+
+[discrete]
+=== Product
+To report a bug, or to suggest new features and enhancements, file an issue in the link:https://github.com/openshift/tektoncd-pipeline-operator[GitHub Project]. Include as many details as possible in the description of the issue.
+
+
+[discrete]
+=== Documentation
+If you find an error or have suggestions for improving the documentation, report an issue in link:https://github.com/openshift/pipelines-docs[pipelines documentation] repository.
+
+[discrete]
+=== Getting in touch with the product team
+For questions and feedback subscribe to the link:mailto:pipelines-interest@redhat.com[pipelines-interest@redhat.com]  mailing list or join the link:https://tektoncd.slack.com/archives/CKWCQD2EP[#openshift-pipelines] channel on Tekton Slack.
+
+== Deprecated Features
+
+The following items are deprecated in 0.8.0, and will be removed in the next release:
+
+* `EventListenerTrigger.interceptor` is now deprecated. Use Interceptors instead.
+* `EventListenerTrigger.binding` is now deprecated. Use `bindings` instead.
+* `status` field is now removed from `Pipeline` and `PipelineResource`. This field will stay in `v1alpha1`, however it will be removed in the next API version.
```

Comment:
If it is going to stay for `v1alpha1` and removed later, should we be listing it here already? Also should this be V1alpha1 with no back ticks?

Reply:
have changed to V1alpha1

Reply:
accepted - change to V1alpha1